version/version.go

```diff
@@ -1,5 +1,5 @@
 package version
 
 var (
-	Version = "0.2.2"
+	Version = "0.2.3"
 )
```

Comment:
It is missing update the tags in the SOP files.

Reply:
> It is missing a few steps regards the SOP. ( see in the readme ).
> Also, IMHO we should not perform the changes made in te CHANGELOG file and it should be kept as before in order to follow the standard.

I'll do that now. Sure, I wasn't aware there was a standard for this (but how we had it was a little bit wrong).

Reply:
Hi @craicoverflow ,

Se that the roles still not pointing to the new tag in the following files.
- https://github.com/aerogear/mobile-developer-console-operator/blob/master/deploy/monitor/prometheus_rule.yaml#L21
- https://github.com/aerogear/mobile-developer-console-operator/blob/master/deploy/monitor/mdc_prometheus_rule.yaml

Reply:
No problem that all.